### PR TITLE
Update language_strings.json

### PR DESCRIPTION
**Pull request by luisrival**
Spanish was added in the languages

### CODE REVIEW

**Comment by DEADSEC-SECURITY** [comment]
@luisrival was this tested? Could list the devices that this PR passed and worked?

**Comment by luisrival** [comment]
It´s working... yes 
as expected..... not shure

It recognize "Si" (Yes) and "No" fine, also I change "custom actions" and " "text": "This is a test of the alexa actions custom skill. Did it work?"  to "acciones personalizadas" and "text": "Esto es una prueba de la skill Notificaciones de alexa, ¿Funcionó?" the translation and it responds correct and in spanish.

But when I run an automation with the notification it doesn't work until I say "acciones personalizadas". I don´t know if it is related with the language or it is an other error.

**Comment by luisrival** [comment]
Hi

Yes it is tested and is working fine. 
What do you mean with "devices that this PR passed and worked"?

**Comment by DEADSEC-SECURITY** [comment]
Like did you use a Echo, FireTV or any other amazon device to test this change? @luisrival

**Comment by luisrival** [comment]
I tested in echo dot, show and it works fine. Also I tested in sonos one and bean connected with alexa, but in these the notification is trigger only when I trigger manually saying "acciones personalizadas" I don´t know if is a languaje problem or the skill or alexa

File: lambda/language_strings.json [hunk 1]
```diff
@@ -14,6 +14,21 @@
 		"STOP_MESSAGE": "Goodbye!",
 		"WELCOME_MESSAGE": "Welcome to {}. Input text is: {}"
 	},
+	"es": {
+	    "ERROR_401": "Parece que no tengo autorización para comunicarme con el asistente de inicio, verifique la vinculación de su cuenta o su token de acceso de larga duración e intente nuevamente",
+		"ERROR_404": "Parece que no puedo encontrar la entidad de texto de entrada. Verifica que lo hayas agregado a Home Assistant e inténtalo de nuevo",
```
Comment (DEADSEC-SECURITY) [inline lambda/language_strings.json:19]: Please fix indentation

**Review by DEADSEC-SECURITY — CHANGES_REQUESTED** [review]
Your missing a file for the new language. Please check the skill-manifests folder for reference and before creating make sure the skill-manifest is according to amazon documentation otherwise it might not recognize the language properly. This should fix the issue you refered.

**Comment by DEADSEC-SECURITY** [comment]
Yup you're just missing the skill manifest for the es language now.

**Comment by DEADSEC-SECURITY** [comment]
Any news?

**Review by DEADSEC-SECURITY — CHANGES_REQUESTED** [review]
Missing skill manifest

**Comment by DEADSEC-SECURITY** [comment]
Due to changes in the skill this PR needs further development to work

**Pull request closed without merging**